README.md

```diff
@@ -49,28 +49,28 @@ We're using [Jest](https://jestjs.io/) and [React Native Testing Library](https:
 ### Adding New Text
 
 1. Add new strings in English to `src/i18n/strings.ftl` using [Fluent syntax](https://projectfluent.org/fluent/guide/), e.g.
-    ```Fluent
-    # Header for a paragraph describing projects
-    ABOUT-PROJECTS = ABOUT
-    # Text describing what projects are
-    projects-description =
-      Projects are really great, probably iNat's best feature.
-    ```
-    Try to match case and strike a balance between specificity and reusability when choosing a key. Please add context comments to help translators understand how the text is used, avoid variables whenever possible, and try to keep `strings.ftl` alphabetized by key.
+   ```Fluent
+   # Header for a paragraph describing projects
+   ABOUT-PROJECTS = ABOUT
+   # Text describing what projects are
+   projects-description =
+     Projects are really great, probably iNat's best feature.
+   ```
+   Try to match case and strike a balance between specificity and reusability when choosing a key. Please add context comments to help translators understand how the text is used, avoid variables whenever possible, and try to keep `strings.ftl` alphabetized by key.
 1. Run `node src/i18n/i18ncli.js build` to build the JSON files i18next needs to access text in the app
 1. In a commponent, use the `useTranslation` hook to reference your new string, e.g.
-    ```jsx
-    import { useTranslation } from "react-i18next";
-    const MyComponent = ( ) => {
-      const { t } = useTranslation();
-      return (
-        <View>
-          <Text>{ t( "ABOUT-PROJECTS" ) }</Text>
-          <Text>{ t( "projects-description" ) }</Text>
-        </View>
-      );
-    };
-    ````
+   ```jsx
+   import { useTranslation } from "react-i18next";
+   const MyComponent = () => {
+     const { t } = useTranslation();
+     return (
+       <View>
+         <Text>{t("ABOUT-PROJECTS")}</Text>
+         <Text>{t("projects-description")}</Text>
```

Comment:
Our convention is to have spaces inside parens, so please revert these two lines to the way the were. The extra \` is a good catch, though.

Reply:
@kueda you can close this PR as I'd like to do it again from a fresh codebase as I encountered issues when trying to push my changes after others were already merged.